tests/integration/targets/connection_nsenter/runme.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+[[ -n "${DEBUG:-}" || -n "${ANSIBLE_DEBUG:-}" ]] && set -x
+
+readonly IMAGE="quay.io/ansible/toolset:latest"
+readonly COLLECTION_ROOT="$(git rev-parse --show-toplevel)"
```

Comment:
Neither is `git` installed, nor is there always a git repository around when tests are run.
```suggestion
readonly COLLECTION_ROOT="$(cd ../../../.. ; pwd)"
```